### ISSUE TITLE
Support 16khz

### ISSUE THREAD

**Issue by thewh1teagle** [yuyun2000/SpeechDenoiser#2]
What's need to be added to support 16khz? is it a limit in the model the we stick to 48khz?

**Comment by yuyun2000** [comment]
Yes, the original model is 48Khz. If you want to process 16Khz audio, you need to resample it yourself before inputting it.

**Comment by thewh1teagle** [comment]
> Yes, the original model is 48Khz. If you want to process 16Khz audio, you need to resample it yourself before inputting it.

Thanks! 
How differ is from the original impl of them? I'm wondering about implementing it in [sherpa-rs](https://github.com/thewh1teagle/vibe) which is also based on onnxruntime

**Comment by yuyun2000** [comment]
我听起来onnx模型的结果和原始的结果差别并不大，但并不是完全一样的；
这个推理代码十分简单，我觉得很容易进行集成；不过据我所知，new kaldi并没有计划支持语音降噪
